README.md

```diff
@@ -12,87 +12,154 @@
 
 # TestResultWebApp (TRWebApp)
 
-::: {.contents local=""}
-:::
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+    -   [How to install](#how-to-install)
+-   [Usage](#usage)
+-   [Contribution](#contribution)
+-   [Sourcecode Documentation](#documentation)
+-   [Feedback](#feedback)
+-   [About](#about)
+    -   [Maintainers](#maintainers)
+    -   [Contributors](#contributors)
+    -   [License](#license)
+
+## Getting Started
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) is a
+generic web based open source application which allows processing
+displaying of test result data in a sophisticated way.
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) uses
+a mysql data base, nodejs on server side and JavaScript on Web-Browser
+side.
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) was
+initially implemented at
+![IMG_BOSCH](packagedoc/additional_docs/pictures/Boschlogo.png) [Robert
+Bosch Car
+Multimedia](https://www.bosch.de/unser-unternehmen/bosch-in-deutschland/hildesheim/)
+and was open sourced in 2020.
 
-## Package Documentation
+### How to install
```

Comment:
Hi Ngoan,
How to install is wrong here.
Somehow we should have at the end a small overview how to get the whole system run.
There was an architecture diagram as I remember. Some steps how to setup I have written in a readme. Can you please look if we can use this information. 
Best would be if this is part of  [TRWebApp's Documentation](https://github.com/test-fullautomation/testresultwebapp/blob/develop/TestResultWebApp/TestResultWebApp.pdf)
Thank you,
 Thomas